airflow/utils/log/file_task_handler.py

```diff
@@ -190,74 +220,22 @@ def _read(self, ti: TaskInstance, try_number: int, metadata: dict[str, Any] | No
                 log = f"*** Failed to load local log file: {location}\n"
                 log += f"*** {str(e)}\n"
                 return log, {"end_of_log": True}
-        elif self._should_check_k8s(ti.queue):
-            try:
-                from airflow.kubernetes.kube_client import get_kube_client
-
-                kube_client = get_kube_client()
+        else:
 
-                log += f"*** Trying to get logs (last 100 lines) from worker pod {ti.hostname} ***\n\n"
+            log += f"*** Log file does not exist: {location}\n"
```

Comment:
```suggestion
            log += f"*** Local log file does not exist, trying to fetch logs from executor environment ***\n\n"
```

This more closely matches what was there previously as well as the new context you added.